serving/src/test/java/feast/serving/it/ServingServiceBigTableIT.java

```diff
@@ -601,6 +614,118 @@ public void shouldReturnCorrectRowCount() {
+            .put("f_int32_list", ValueProto.ValueType.Enum.INT32_LIST)
+            .put("f_float_list", ValueProto.ValueType.Enum.FLOAT_LIST)
+            .put("f_double_list", ValueProto.ValueType.Enum.DOUBLE_LIST)
+            .put("f_string_list", ValueProto.ValueType.Enum.STRING_LIST)
+            .put("f_bytes_list", ValueProto.ValueType.Enum.BYTES_LIST)
+            .put("f_bool_list", ValueProto.ValueType.Enum.BOOL_LIST)
+            .build();
+
+    TestUtils.applyFeatureTable(
+        coreClient, "default", "all_types", ImmutableList.of("entity"), allTypesFeatures, 7200);
+
+    Schema schema =
+        SchemaBuilder.record("AllTypesRecord")
+            .namespace("")
+            .fields()
+            .requiredLong("f_int64")
+            .requiredInt("f_int32")
+            .requiredFloat("f_float")
+            .requiredDouble("f_double")
+            .requiredString("f_string")
+            .requiredBytes("f_bytes")
+            .requiredBoolean("f_bool")
+            .name("f_int64_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().longType()))
+            .noDefault()
+            .name("f_int32_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().intType()))
+            .noDefault()
+            .name("f_float_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().floatType()))
+            .noDefault()
+            .name("f_double_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().doubleType()))
+            .noDefault()
+            .name("f_string_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().stringType()))
+            .noDefault()
+            .name("f_bytes_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().bytesType()))
+            .noDefault()
+            .name("f_bool_list")
+            .type(SchemaBuilder.array().items(SchemaBuilder.builder().booleanType()))
+            .noDefault()
+            .endRecord();
+
+    GenericData.Record record =
+        new GenericRecordBuilder(schema)
+            .set("f_int64", 10L)
+            .set("f_int32", 10)
+            .set("f_float", 10.0)
+            .set("f_double", 10.0D)
+            .set("f_string", "test")
+            .set("f_bytes", ByteBuffer.wrap("test".getBytes()))
+            .set("f_bool", true)
+            .set("f_int64_list", ImmutableList.of(10L))
+            .set("f_int32_list", ImmutableList.of(10))
+            .set("f_float_list", ImmutableList.of(10.0))
+            .set("f_double_list", ImmutableList.of(10.0D))
+            .set("f_string_list", ImmutableList.of("test"))
+            .set("f_bytes_list", ImmutableList.of(ByteBuffer.wrap("test".getBytes())))
+            .set("f_bool_list", ImmutableList.of(true))
+            .build();
+
+    ValueProto.Value entity = DataGenerator.createStrValue("key");
+
+    ingestData(
+        "all_types",
```

Comment:
nit: Should we have a new variable for this since its used [here](https://github.com/feast-dev/feast-java/pull/25/files#diff-d6874d7d4f27250d965a1bb4013134238cdf6ac7b97b38f618181577f61b52edR717) as well.